### ISSUE TITLE
Would something like this be possible

### ISSUE THREAD

**Issue by kougan** [kyranb/SoundCloud-Feed-Cleaner#2]
Hey great extension, it works fantastic, didn't know how to contact you, but I just wanted to ask to see if something was possible and how hard would it be to implement it. It'd be great if you could filter which reposts get removed. Say an artist called "Example" reposted a song(called something like Song1 by Artist1 & Example) in which he contributed, but hasn't uploaded it on his SoundCloud. So that would be a repost that we would still see on Uploads Only. If the song title has the name of the artist that reposted the song it doesn't count it as a repost, but if an artist reposts a song that doesn't have his name we don't see it

**Comment by kyranb** [comment]
Hi there, glad you found it useful and managed to figure out for to install
it.

May I ask how you came across it?

I completely understand your frustration and the functionality you are
after. It's something I've definitely experienced myself when an artist
shares a remix, collaboration, DJ set, or some kind of exclusive first play
on another account.

Have you written any JavaScript yourself? It should be pretty trivial to
implement, however if you are willing to wait a little bit I'll work in the
functionality you described :)

**Comment by kougan** [comment]
A friend of mine found it actually, and I have a small experience with JavaScript, but if you can implement it it'd be great x)

**Comment by kyranb** [comment]
Glad he/she found it and it has provided some use. I'm surprised others Soundcloud users don't get frustrated with a cluttered stream as well. I spent a while looking for a settings option, or other hack solution before coming up with my own.

Anyway, feel free to checkout this commit: 949171ca35f282a9290241da74eb42d719d25e13

I've added the functionality you requested and it appears to be working fine for me (Latest version of Chrome on OSX). It currently only works if the artist name is the same (case insensitive). 

Thanks for suggestion as I listen to a lot of Electronic Music and artists regularly repost their music on another channel.
